### ISSUE TITLE
zstd / openzfs 2.0.3?

### ISSUE THREAD

**Issue by mailinglists35** [openzfsonwindows/ZFSin#333]
Hi,

I'd like to start using zfs on a secondary hdd

Do the nighty builds contain openzfs v2 code, or there are still issues to merge the openzfs v2 release? (for example I searched for zstd and see no results).

Also, what is the openzfs/zol version equivalent of current release? I only see some nordic-language names and dates in the releases section :)

Thanks

**Comment by lundman** [comment]
ZFSin is closer to 0.8.x branch IIRC, its been a while tho, so I don't remember exactly.
The OpenZFS repo (https://github.com/openzfsonwindows/openzfs/tree/windows) is from around 2.0 release, and has zstd in it.

**Comment by mailinglists35** [comment]
Oh, I'm even more confused now learning about two windows repos; what is the difference between this and the other repo? :)

**Comment by DanielSmedegaardBuus** [comment]
Yeah, that's the same kinda question I came here with, and I'm also confused now :D Well, I was confused before as well, because the release tag versions are all 0.xx... So that's very, very far from anything remotely current in OpenZFS, and clearly not the same, because we're at least at 1.9.x here... So how do we know which version ZFS for Windows is running? I don't have `man zpool/zfs` available, and `--version` isn't an argument that zfs or zpool understands, so...

Where exactly is ZFS on Windows, version-wise, and any idea when there'll be 2.x, or the equivalent of, that'll allow us to play with zstd?

This would be great to have, either as a wiki thing, or even better, just knowing which version it is under the releases page.

Thanks :)

**Comment by lundman** [comment]
The first port (ZFSin) was a proof-of-concept to see if it could work. Like other platforms, it had its own version.  Took a snapshot around 0.8 (ZOL) days on OsX (1.9.x) and started working on it. But did not want to call it 1.x - since that *could* be taken as somewhat stable (version 1 vs, not-even-one-yet).

However, all the various versions, and inconsistent feature sets, is not a good thing.

So OpenZFS decided we should put in a massive effort and unify the repository.

ZOL was renamed into OpenZFS.
FreeBSD has already been merged into OpenZFS.
macOS has a PR in OpenZFS to be merged. (test 2.0.x build is available)
and Windows port was restarted to be based on the OpenZFS-2.0 branch, and
will soon become a PR to it. (test 2.0.x build is available)

Going forward, there is one version system. One set of features. 

Most importantly, it will be just a matter of rebasing against OpenZFS to get latest features, bug fixes and code.

It has taken me quite a long time to get the OpenZFS repo to the same quality as ZFSin, but it is nearly there. Mostly just the same code re-arranged, but done properly, so it has a future and can be maintained.

But yes, there is this period of transition, which is confusing.

**Comment by DanielSmedegaardBuus** [comment]
Thank you for the elaboration! And that's pretty cool about a 2.0 test build for Windows :) Windows is my gaming OS, and I have some stuff I'd like to try out in that regard with ZFS and zstd at max compression. No time like the present! :D Cheers!

**Comment by Tsaukpaetra** [comment]
Oh hey, just saw this. Guess I'll wait to see when betas start, eh?

**Comment by Ambyjkl** [comment]
Looking forward to this. Maybe one day we'll be able to install Windows on ZFS. That would be the end game.
